Dockerfile

```diff
@@ -161,12 +175,14 @@ COPY scripts /scripts
 RUN curl -sL https://github.com/jwilder/dockerize/releases/download/v0.6.1/dockerize-linux-amd64-v0.6.1.tar.gz | tar zxf - -C /bin/
 
 # Update access of required directories to allow not running in root for openshift
-RUN chmod -R g+rw ${CATALINA_HOME}/logs  && \
-    chmod -R g+rw ${CATALINA_HOME}/lib  && \
-    chmod -R g+rw ${CATALINA_HOME}/work  && \
-    chmod -R g+rw ${CATALINA_HOME}/conf  && \
-    chmod -R g+rw ${CATALINA_HOME}/bin  && \
-    chmod -R g+rw ${CATALINA_HOME}/webapps && \
+RUN chmod -R 777 ${CATALINA_HOME}/logs  && \
+    chmod -R 775 ${CATALINA_HOME}/lib  && \
+    chmod -R 775 ${CATALINA_HOME}/work  && \
+    chmod -R 775 ${CATALINA_HOME}/conf  && \
+    mkdir -p ${CATALINA_HOME}/conf/path && \
```

Comment:
Why are we creating this `$CATALINA_HOME}/conf/path` directory?

Reply:
Changed it to reflect reality